programmers/수박수박수박수.js

```diff
@@ -0,0 +1,11 @@
+function solution(n) {
+    let watermelon = ``;
+    for(let i = 1; i <= n; i++) {
+        if (i % 2 === 0) {
```

Comment:
쓸데없이 1을 더하거나 빼는 상황이 아니라면 메모리 구조상 <= 보다는 < 가 낫다.
=== 0 생략가능하다.
```suggestion
    for(let i = 1; i < n; i++) {
        if (i % 2) {
```